CVE-2019-25071.json

```diff
@@ -0,0 +1,80 @@
+{
```

Comment:
Expected location (2019/25xxx/CVE-2019-25071.json) of file does not match actual location (CVE-2019-25071.json)